src/caret_analyze/architecture/architecture_loaded.py

```diff
@@ -1588,6 +1602,7 @@ class CallbackPathSearched():
     def __init__(
         self,
         node: NodeStruct,
+        max_callback_construction_order_on_path_searching: int
```

Comment:
I suggest changing `max_callback_construction_order_on_path_searching` here to `max_callback_construction_order` because it is obvious that this is related to path searching. (Sorry for changing my previous suggestion.)

Reply:
I agree.